esrally/metrics.py

```diff
@@ -130,6 +130,11 @@ def __init__(self, cfg):
         secure = self._config.opts("reporting", "datastore.secure") == "True"
         user = self._config.opts("reporting", "datastore.user")
         password = self._config.opts("reporting", "datastore.password")
+        # poor man's boolean conversion
+        verify_certs = self._config.opts("reporting", "datastore.verify_certs", default_value="True", mandatory=False) == "True"
```

Comment:
I'd rather use `datastore.certs.verify` as name (in the config file we usually use dots as a separator).